web/store/issue/project/issue.store.ts

```diff
@@ -223,6 +223,12 @@ export class ProjectIssues extends IssueHelperStore implements IProjectIssues {
 
       const response = await this.createIssue(workspaceSlug, projectId, data);
 
+      if (data.cycle_id && data.cycle_id !== "")
```

Comment:
This seems to be only for project issues, but looks like it needs to be done in project views store, draft issues store, archived issue store etc, wherever we allow to be grouped by cycle and module

Reply:
Done! I've implemented the changes wherever necessary.